src/components/dialogs/line-creation-dialog.js

```diff
@@ -234,123 +276,213 @@ const LineCreationDialog = ({
         handleClose();
     };
 
+    const handleSelectionChange = (element) => {
+        let msg;
+        return fetchEquipmentInfos(
+            studyUuid,
+            selectedNodeUuid,
+            'lines',
+            element.id,
+            true
+        ).then((response) => {
+            if (response.status === 200) {
+                response.json().then((line) => {
+                    setFormValues(null);
```

Comment:
why are you doing this before calling `setFormValues` with `lineFormValues` ?

Reply:
fixed